impl/src/main/java/org/jboss/cdi/tck/tests/extensions/configurators/bean/BeanConfiguratorTest.java

```diff
@@ -60,38 +65,51 @@ public void testCreationalAndDisposalMethods() {
         Bean<Skeleton> skeletonBean = getUniqueBean(Skeleton.class, Undead.UndeadLiteral.INSTANCE);
         CreationalContext<Skeleton> skeletonCreationalContext = getCurrentManager().createCreationalContext(skeletonBean);
         Skeleton skeleton = skeletonBean.create(skeletonCreationalContext);
-        
+
         Bean<Zombie> zombieBean = getUniqueBean(Zombie.class, Undead.UndeadLiteral.INSTANCE, Dangerous.DangerousLiteral.INSTANCE);
         CreationalContext<Zombie> zombieCreationalContext = getCurrentManager().createCreationalContext(zombieBean);
         Zombie zombie = zombieBean.create(zombieCreationalContext);
-        
+
+        // instantiate Ghost and Vampire to verify their creational methods were called
+        Bean<Ghost> ghostBean = getUniqueBean(Ghost.class, Undead.UndeadLiteral.INSTANCE);
+        CreationalContext<Ghost> ghostCreationalContext = getCurrentManager().createCreationalContext(ghostBean);
+        Ghost ghost = ghostBean.create(ghostCreationalContext);
+
+        Bean<Vampire> vampireBean = getUniqueBean(Vampire.class, Undead.UndeadLiteral.INSTANCE);
+        CreationalContext<Vampire> vampireCreationalContext = getCurrentManager().createCreationalContext(vampireBean);
+        Vampire vampire = vampireBean.create(vampireCreationalContext);
```

Comment:
For Ghost and Vampire - yes, agree. For Skeleton and Zombie - not really. The reason is that I need a reference to their respective `CreationalContext` and their `Bean<T>` in order to call `destroy` methods.